web/sites/default/default.services.yml

```diff
@@ -93,6 +93,21 @@ parameters:
     # Disabling the Twig cache is not recommended in production environments.
     # @default true
     cache: true
+    # File extensions:
```

Comment:
@RichardDavies , I believe this needs to be added to the following:

- web/sites/default/local.services.yml
- web/sites/default/services.pantheon.preproduction.yml
- web/sites/default/services.yml

I would also expect some config changes to text formats similar to Kevin's PR for portlandor: https://github.com/eGovPDX/portlandor/pull/2367/files

Reply:
@kxwang , the PR for portlandor only updated the services.yml, does that load for local and pantheon as well with only the overrides coming from the local.services.yml and services.pantheon.preproduction.yml? It seems to work fine in my local and certainly worked in the multidev without those allowed extensions getting added to the other files.

CC @RichardDavies

Reply:
My understanding is that the services YAML files for Local and Pantheon overrides anything that's set in services.yml. For PGOV, the `web/sites/default/services.pantheon.preproduction.yml` file only contain comments and we don't have the file `web/sites/default/services.pantheon.production.yml`.

Reply:
Yes, we only need to add the twig extensions to `services.yml` because the other YAMLs are only additions/overrides to what's in `services.yml`.

And as for the text format changes from Kevin's PR that are "missing" here, I think those are from a prior Drupal update or something else. This update was only a Twig fix and the employee text format configs already appear to contain the changes from Kevin's PR.

Reply:
Yes, looks like the editor changes were from 9.4.5, which Employees had previously updated to already. https://www.drupal.org/project/drupal/releases/9.4.5